crates/wasmcloud-host/src/host.rs

```diff
@@ -376,8 +401,8 @@ impl Host {
     }
 
     /// Updates a running actor with the bytes from a new actor module. Invoking this method will
-    /// cause all messages inbound to the actor to block and it can take a second or two depending
-    /// on the size of the actor and the underlying engine you're using (e.g. JIT vs. interpreter).
+    /// cause all pending messages inbound to the actor to block while the update is performed. It can take a few
+    /// seconds depending on the size of the actor and the underlying engine you're using (e.g. JIT vs. interpreter).
     /// ## ⚠️ Caveats
     /// * Take care when supplying a value for the `new_oci_ref` parameter. You should be consistent
     /// in how you supply this value. Updating actors that were started from OCI references should
```

Comment:
Unfortunately I can't comment on it as it wasn't changed in this PR, but for the snippet a few lines down:
```
    /// * You should only ever use this method when running in standalone/isolated mode. If you have a control
    /// interface configured, then you should perform live updates by using a control interface client
    /// and a suitable OCI reference URL.
```
What are the consequences of this?